docs/greek.rst

```diff
@@ -262,12 +261,90 @@ For example to use upper case vowels with rough breathing and an acute accent:
 
 Accents indicate the pitch of vowels. An *acute accent* or *ὀξεῖα (oxeîa)* indicates a rising pitch on a long vowel or a high pitch on a short vowel, a *grave accent* or *βαρεῖα (bareîa)* indicates a normal or low pitch and a *circumflex* or *περισπωμένη (perispōménē)* indicates high or falling pitch within one syllable.
 
-Breathings, which are used not only on vowels, but also on *ρ*, indicate the presence or absence of a voiceless glottal fricative - rough breathing indicetes a voiceless glottal fricative before a vowel, like in *αἵρεσις (haíresis)* and smooth breathing indicates none.
+For accessing the accents and numerals:
+
+.. code-block:: python  
+
+    In[1]: from cltk.corpus.greek.alphabet import ACCENTS, NUMERAL_SIGNS
+    In [2]:  print(ACCENTS)
+    Out[2]: ['ͺ', '΄', '΅', '·', '᾽', 'ι', '᾿', '῀', '῁', '῍', '῎', '῏', '῝', '῞', '῟', '῭', '΅', '`', '´', '῾']
+    In [3]: print(NUMERAL_SIGNS)
+    Out[7]: ['ʹ', '͵']
```

Comment:
Do I understand correctly that this snippet is not run on Travis CI? See https://github.com/cltk/cltk/pull/591 for background.